onair/src/run_scripts/execution_engine.py

```diff
@@ -78,25 +82,21 @@ def parse_configs(self, config_filepath):
             ## Parse Required Data: Names
             self.parser_file_name = config['DEFAULT']['ParserFileName']
 
-            ## Parse Required Data: Plugin name to path dict
-            config_plugin_list = config['DEFAULT']['PluginList']
-            ast_plugin_list = self.ast_parse_eval(config_plugin_list)
-            if isinstance(ast_plugin_list.body, ast.Dict) and len(ast_plugin_list.body.keys) > 0:
-                temp_plugin_list = ast.literal_eval(config_plugin_list)
-            else:
-                raise ValueError(f"{config_plugin_list} is an invalid PluginList. It must be a dict of at least 1 key/value pair.")
-            for plugin_name in temp_plugin_list.values():
-                if not(os.path.exists(plugin_name)):
-                    raise FileNotFoundError(f"In config file '{config_filepath}', path '{plugin_name}' does not exist or is formatted incorrectly.")
-            self.plugin_list = temp_plugin_list
+            ## Parse Required Data: Plugins
+            self.knowledge_rep_plugin_dict = self.parse_plugins_dict(config['DEFAULT']['KnowledgeRepPluginDict'])
+            self.learners_plugin_dict = self.parse_plugins_dict(config['DEFAULT']['LearnersPluginDict'])
+            self.planners_plugin_dict = self.parse_plugins_dict(config['DEFAULT']['PlannersPluginDict'])
+            self.complex_plugin_dict = self.parse_plugins_dict(config['DEFAULT']['ComplexPluginDict'])
+
+            ## Parse Optional Data: Flags
+            ## 'RUN_FLAGS' must exist, but individual flags return False if missing
+            self.IO_Flag = config['RUN_FLAGS'].getboolean('IO_Flag')
```

Comment:
I'm very ready for these run flags to go away.

Reply:
They can easily be removed, if that is what we want to do. Are they no longer of any use?

Reply:
Looks like we already have an issue for this: https://github.com/nasa/OnAIR/issues/60